app/decorators/case_contact_decorator.rb

```diff
@@ -70,4 +73,8 @@ def medium_type_icon
       object.medium_type
     end
   end
+
+  def notes
+    object.notes ? object.notes.truncate(NOTES_WORD_LIMIT) : ""
```

Comment:
You can actually simplify this even more by:

```
def notes
  object.notes.to_s.truncate(NOTES_WORD_LIMIT)
end
```

The truncate method will fail for `nil`, but `nil.to_s` is an empty string, so that ensures you are always applying the method to a string.

Reply:
Ooh awesome! Updated!